### PR TITLE
add decorate PR to git scm

### PR DESCRIPTION
**Pull request by lusol**
add decorate function for new PR field. Uses github api and breaker to retrieve the url.
related pr: https://github.com/screwdriver-cd/screwdriver/issues/822

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -1628,15 +1628,18 @@ jobs:
 
         it('returns a pull request with the given prNum', () => {
             githubMock.pullRequests.get.yieldsAsync(null,
-                { number: 1, head: { sha } }
+                { html_url: "https://github.com/'repoOwner/repoName'/pull/1",
```
Comment (tkyi) [inline test/index.test.js]: can you fix indentation so it looks more like:
```
{
    html_url: "https://github.com/'repoOwner/repoName'/pull/1",
    number: 1,
    head: { sha }
}
```

Comment (tkyi) [reply test/index.test.js]: Why is `'repoOwner/repoName'` in quotes?

Comment (lusol) [reply test/index.test.js]: for the first point, lint won't let me continue if I don't format it the way above...

Comment (tkyi) [reply test/index.test.js]: Weird :[

**Pull request merged**